lib/frontend/post-ghae-complete.js

```diff
@@ -22,38 +22,49 @@ module.exports = function (opts) {
 
     // check valid referer, needed to validate ghaehost
     if (!req.get('referer')) {
-      handleExceptions(res, 400, 'Request not coming from valid referer');
-      return;
+      console.log(`Request not coming from valid referer ${JSON.stringify(query)} - request ip ${req.ip}`);
+      return res.status(400).render('ghae-register-error.hbs', {
+        error: 'Request not coming from valid referer',
+      });
     }
     // Check state & code present in params
     if (!state) {
-      handleExceptions(res, 400, 'Missing Auth state parameter');
-      return;
+      console.log(`Missing Auth state parameter ${JSON.stringify(query)} - request ip ${req.ip}`);
+      return res.status(400).render('ghae-register-error.hbs', {
+        error: 'Missing Auth state parameter',
+      });
     }
     if (!code) {
-      handleExceptions(res, 400, 'Missing OAuth Code');
-      return;
+      console.log(`Missing OAuth Code ${JSON.stringify(query)} - request ip ${req.ip}`);
+      return res.status(400).render('ghae-register-error.hbs', {
+        error: 'Missing OAuth Code',
+      });
     }
     // retrive ghaeHost from state
     let registration = await Registration.getRegistration(state);
     // check valid state or not
     if (!registration || !lessThanOneHourAgo(registration.createdAt)) {
       console.log(`Invalid Auth state parameter ${JSON.stringify(query)} - request ip ${req.ip}`);
-      handleExceptions(res, 401, 'Invalid Auth state parameter');
-      return;
+      return res.status(401).render('ghae-register-error.hbs', {
+        error: 'Invalid Auth state parameter',
+      });
     }
     let ghaeHost = parseHost(req.get('referer'));
     let githubHost = registration.githubHost;
     // check valid ghaehost
     if (githubHost !== ghaeHost) {
-      handleExceptions(res, 401, 'Request coming from invalid host');
-      return;
+      console.log(`Request not coming from valid GitHub AE instance ${JSON.stringify(query)} - request ip ${req.ip}`);
```

Comment:
can you replace console.log with 

const { getLog } = require('../../lib/config/logger');
let logger = getLog();
logger.error